packages/cli-repl/src/cli-repl.ts

```diff
@@ -605,6 +626,19 @@ export class CliRepl implements MongoshIOProvider {
     );
   }
 
+  async setLoggingEnabled(enabled: boolean): Promise<void> {
```

Comment:
do we want to consider `disableLogging()` and `enableLogging()` shorthands similar to telemetry?

Reply:
We only have those because they predate the `config` object, otherwise we probably wouldn't